x/gov/keeper/proposal.go

```diff
@@ -43,11 +52,22 @@ func (keeper Keeper) SubmitProposal(ctx context.Context, messages []sdk.Msg, met
 	for _, msg := range messages {
 		msgsStr += fmt.Sprintf(",%s", sdk.MsgTypeURL(msg))
```

Comment:
Is there a reason for keeping this here?

Reply:
You mean `msgsStr += fmt.Sprintf(",%s", sdk.MsgTypeURL(msg))` ?
I don't see why we should remove it with the view of minimizing diffs with upstream

Reply:
My bad I thought that was a change on your side

Reply:
Ok, thanks. Will resolve this convo